src/charts.js

```diff
@@ -305,22 +319,44 @@ function dateToHumanString(date) {
 }
 
 function downloadsURL(pkg, from, to) {
-  return '/downloads/range/' + dateToString(from) + ':' + dateToString(to) + '/'
+  return 'http://npm-stat.com/downloads/range/' + dateToString(from) + ':' + dateToString(to) + '/'
```

Comment:
Just saw your commit message. This makes debugging a lot easier, so it's okay for me.